pkg/common/ac_operations.go

```diff
@@ -278,15 +279,21 @@ func (a *ACOperationsImpl) tryToFindAC(acs []*accrd.AvailableCapacity, storageCl
 	var (
 		allocatedCapacity int64 = math.MaxInt64
 		foundAC           *accrd.AvailableCapacity
+		driveUUID         = make([]string, 0)
 	)
 	for _, ac := range acs {
+		// Available capacity with system disk location won't be allocated
+		if len(driveUUID) == 0 {
+			driveUUID = a.k8sClient.GetSystemDriveUUID(context.Background(), ac.Spec.NodeId)
```

Comment:
Maybe we can try to move this code outside of the loop? Can we retrieve system drives for all nodes with single request?

Reply:
I removed this code due to this comment https://github.com/dell/csi-baremetal/pull/95#discussion_r498657251